packages/components/src/components/Select/Combobox.tsx

```diff
@@ -135,14 +168,15 @@ export const SelectCombobox = (
                             {clearButton}
                         </RadixSlot>
                     )}
+
                     <button
+                        {...getToggleButtonProps()}
```

Comment:
reordering this might cause issues because overriding some props will be overridden that should not be

Reply:
will revert